devTools/Dockerfile-php80-xdebug

```diff
@@ -25,8 +20,16 @@ RUN set -eux; \
     apk add --no-cache --virtual .phpexts-rundeps ${runDeps}; \
     apk del .build-deps
 
+RUN apk add --no-cache \
```

Comment:
Maybe for Infection it's ok but usually exactly this part is persistent and does not change very often.

p.s. I'd leave it right after `FROM`.

Reply:
Well, I had to change it several times already, and I can say that a build for Xdebug takes more than few moments.

Reply:
fine with me.
Don't expect that we will install other php extensions in the nearest feature.